undo-propose.el

```diff
@@ -181,6 +186,33 @@ buffer contents are copied."
   (interactive)
   (ediff-buffers undo-propose-parent (current-buffer)))
 
+(defvar undo-propose-marker-list nil
+  "List of markers to update after running undo-propose.")
+
+(defun undo-propose-copy-markers ()
+  "Copy markers registered in `undo-propose-marker-list'."
+  (setq-local undo-propose-marker-map
+              (cl-loop for orig-marker in undo-propose-marker-list
+                       if (eq (marker-buffer orig-marker) undo-propose-parent)
```

Comment:
IMO, if we replace `orig-marker` with `(symbol-value orig-marker)` from line 196 to 200, then user can add any markers to `undo-propose-marker-list` like `(add-to-list 'undo-propose-marker-list 'org-clock-hd-marker).